tensorflow-framework/src/main/java/org/tensorflow/framework/optimizers/Optimizer.java

```diff
@@ -15,21 +15,19 @@
  */
 package org.tensorflow.framework.optimizers;
 
+import java.util.*;
```

Comment:
Where'd this star import come from?

Reply:
Not sure, but it passes style checks.

Reply:
Well we should fix that because they are prohibited under the Google Java Style (https://google.github.io/styleguide/javaguide.html#s3.3-import-statements).

Reply:
Huh, it's not applying to generated files.  I'm going to make another PR for that.

Reply:
That file isn't generated though, I wrote that file.

Reply:
I mixed up `Ops` and `Optimizers`, that's even weirder.  Let me look.

Reply:
Huh, apparently it's not supported by the formatter, that's annoying.  See https://github.com/google/google-java-format/issues/113.  I'll see if I can find a way to do it via spotless.

Reply:
Fixed it manually for now.